src/routes/Router.tsx

```diff
@@ -36,6 +35,12 @@ const RouteWithSubRoutes = (route: TRoutesWithSubRoutes) => {
 };
 
 const Router: FC = () => {
+    const location = useLocation();
```

Comment:
can we make a new hook here useHandleRouteChange?

Reply:
done. please check again. not sure for now we need to add more to it or not